README.md

```diff
@@ -60,9 +61,15 @@ use the `format` option in the parameters you pass to any method.
 format : object|json|array (default:object)
 ```
 
+## Twitter API Versions
+
+To use Twitter API v2 please ensure you set `TWITTER_API_VERSION` to `2`. The package uses API version `1.1` by default.
```

Comment:
Is it not possible to call the v2 endpoints with this set to 1.1.? THat would be ideal.

Reply:
Ahh, it can be switched with `forApiV2()` or `forApiV1()`.

Maybe we should write something like. 

```md
## Twitter API Versions

To set the default twitter API version to v2 instead of the default `v1.1`, set the  `TWITTER_API_VERSION` to `2` in your `.env`.

If you have set the `v1.1` api as the default, you can use use `Twitter::forApiV2()` to get an instance of the v2 client.
The same goes for getting a `v1` instance from a `v2` client, using `Twitter::forApiV1()`.

It is safe to call `Twitter::forApiV1()` on a either a `v1` or `v2` client instance.

```

Reply:
Makes sense. Updated 👍🏾